pandas/core/frame.py

```diff
@@ -3960,52 +3959,40 @@ def f(i):
 
     def _combine_series(self, other, func, fill_value=None, axis=None,
                         level=None, try_cast=True):
+        if fill_value is not None:
+            raise NotImplementedError("fill_value %r not supported." %
```

Comment:
is this hit in tests?

Reply:
can you use .format() here instead

Reply:
Yes to both.  tests.frame.test_operators.TestDataFrameOperators.test_arith_flex_frame:

```
        with tm.assert_raises_regex(NotImplementedError, 'fill_value'):
           self.frame.add(self.frame.iloc[0], fill_value=3)
```